components/dashboards/template-selector/import-selector/component.js

```diff
@@ -0,0 +1,86 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import debounce from 'lodash/debounce';
+import TetherComponent from 'react-tether';
+import { Router } from 'routes';
+import { toastr } from 'react-redux-toastr';
+
+
+class HeaderTopics extends PureComponent {
+  static propTypes = {
+    topics: PropTypes.array.isRequired,
+    setDropdownOpened: PropTypes.func.isRequired
+  };
+
+  toggleDropdown = debounce((bool) => {
+    this.props.setDropdownOpened({ topics: bool });
+  }, 50)
+
+
+  state = { isOpen: false }
+
+
+  handleTopics = ({ id }) => {
+
+    toastr.confirm('Are you sure you want to copy this topic?', {
+      onOk: () => {
+        fetch(`${process.env.WRI_API_URL}/topics/:${id}/clone-dashboard`, {
```

Comment:
@mluena the endpoint is not working (404), I think you need to remove the `:` next to the id param.